app/Locale/German.php

```diff
@@ -445,8 +445,8 @@ public function get_locale(): array
 			'UPLOAD_IMPORT' => 'Importieren',
 			'UPLOAD_IMPORT_SERVER' => 'Importieren von Server',
 			'UPLOAD_IMPORT_SERVER_FOLD' => 'Der Ordner ist leer oder enthält keine lesbaren Dateien zum Verarbeiten. Schauen Sie bitte ins Protokoll (Einstellungen/Protokoll ansehen).',
-			'UPLOAD_IMPORT_SERVER_INSTR' => 'Diese Aktion wird alle Fotos, Ordner und Unterordner importieren, die sich in folgendem Verzeichnis befinden.',
-			'UPLOAD_ABSOLUTE_PATH' => 'Absoluter Pfad zum Verzeichnis',
+			'UPLOAD_IMPORT_SERVER_INSTR' => 'Import all photos, folders and sub-folders located in the folders with the following absolute paths (on server). Paths are space separated, use \\ to escape a space in a path.',
+			'UPLOAD_ABSOLUTE_PATH' => 'Absolute path to directories, space separated',
```

Comment:
```suggestion
			'UPLOAD_IMPORT_SERVER_INSTR' => 'Importiert alle Fotos, Ordner und Unterordner in den folgenden absoluten Pfaden (auf dem Server). Mehrere Pfade können mit Leerzeichen getrennt werden; mit \\ können Sie ein Leerzeichen im Pfad verwenden.',
			'UPLOAD_ABSOLUTE_PATH' => 'Absolute Pfade zu Verzeichnissen, mit Leerzeichen getrennt',
```

I corrected the spelling mistake "Unterordernern" --> "Unterordner".

Moreover, I recognized that we use the formal form to address a user "Sie" (for the Frenchies: vous) instead of the informal form "Du" ("tu"). I know we are very inconsistent about that. This inconsistency has been bothering me for quite a while now, but I haven't felt like fixing that on a global scale.

But we should establish a rule what we want to use and then stick to it:

 - Option 1: Don't address the user directly at all and resort to different alternatives like passive voice, etc. This avoids the problem for languages which have more than a single "you" for everything. Moreover we avoid the problem of deciding whether the user is male or female. (Again for languages which differentiates this aspect.) Something like
   
   > Do you want to delete the photo? This cannot be undone.
   
   would become
   
   > Shall the photo be deleted? This cannot be undone.
   
   Pro: No problem with "Du"/"tu" vs. "Sie"/"vous"; no problem with gender. Con: Passive voice sometimes leads the less legible phrases.

 - Option 2a: Directly address the user in a formal way. Use "Sie"/"vous". 
 
 - Option 2b: Directly address the user in an informal way. Use "Du"/"tu".

Reply:
> Pro: No problem with "Du"/"tu" vs. "Sie"/"vous"; no problem with gender. Con: Passive voice sometimes leads the less legible phrases.

In the same style, my PhD Supervisor wrote his full PhD in passive voice (:scream:). Personally, I went for the "we" as _the authors of the publications_ rather than "we" as _the reader and the author going through the manuscript_.

That being said, I would be going for option 2. Most confirmation windows are directed at the user. e.g. in Debian when doing `apt dist-upgrade`: `Do you want to continue? [Y/n]`

Reply:
That is fine for English. But there was an option 2a vs. 2b. Formal or informal?

Reply:
FYI, the French translations are making use of the Formal form.